tests/WP_SQLite_Translator_Tests.php

```diff
@@ -278,12 +278,13 @@ public function testShowCreateTable1() {
 		$results = $this->engine->get_query_results();
 		# TODO: Should we fix mismatch with original `option_value` text NOT NULL,` without default?
 		$this->assertEquals(
-			"CREATE TABLE _tmp_table (
-	`ID` bigint PRIMARY KEY AUTO_INCREMENT NOT NULL,
```

Comment:
Why do we need to remove PRIMARY KEY from tests?

I tried reverting tests and they failed when `PRIMARY KEY`. Is this expected?

Reply:
The reason for this is because the change i made moves the primary key constraint away from the column definition. The primary key constraint can now be found amongst the other key definitions. In this case:

```mysql
CREATE TABLE `_tmp_table` (
	`ID` bigint NOT NULL AUTO_INCREMENT,
	`option_name` varchar(255) DEFAULT '',
	`option_value` text NOT NULL DEFAULT '',
	PRIMARY KEY (`ID`), -- Primary key has been moved here.
	KEY `_tmp_table__composite` (`option_name`, `option_value`),
	UNIQUE KEY `_tmp_table__option_name` (`option_name`)
```

You could add PRIMARY KEY on the column level if the table has a single column PK but for consistency, I now always move it below column definitions. This also prevents us from having to write separate logic for single and composite PKs.

Reply:
But why are the old tests breaking? I'm concerned that this PR will break existing support in favor of adding the new feature.

Is there a way for us to have both your new tests and the old test working?

Reply:
If i understand your point correctly, you only reverted the tests and then ran them against the new code and then noticed that the tests did not work?

Also, if i understand correct, you want the output of the create table statement to be the same as it was before?

So that means:

1. Not using backticks for keywords, column and table names.
2. Placing the PRIMARY KEY clause next to the column, but only if there is a single column PK
3. Keeping the UNIQUE KEY that SQLIte adds for PKs (because it does not support uniqueness of PKs without it).

Is this correct?

Reply:
> If i understand your point correctly, you only reverted the tests and then ran them against the new code and then noticed that the tests did not work?

Yes.

> Also, if i understand correct, you want the output of the create table statement to be the same as it was before?

No.

I tested this now by adding back an old test to your PR and it passed, so we are good.
Would it make sense to add this test?

```
	public function testShowCreateTableWithColumnKeys() {
		$this->assertQuery(
			"CREATE TABLE _tmp_table (
				`ID` bigint PRIMARY KEY AUTO_INCREMENT NOT NULL,
				`option_name` varchar(255) DEFAULT '',
				`option_value` text NOT NULL DEFAULT '',
				KEY _tmp_table__composite (option_name, option_value),
				UNIQUE KEY _tmp_table__option_name (option_name)
			);"
		);
	}
```

Reply:
Thanks for clarifying and providing an additional unit test. It makes sense now.

I've added the test so we can be sure that create statements with PKs at the column level still works as expected.